Client/src/Remote/RemoteHelper.tsx

```diff
@@ -42,6 +48,14 @@ const fetchRemote = async (url: string, method: string, data: any = {}): Promise
     }
 
     const message = await response.json()
+
+    if (response.status.toString().charAt(0) == '2' && window.location.href.indexOf("/sign-up") > -1) {
+      console.log("sign up successful");
```

Comment:
I dont think we need the console here

Reply:
@GD-Dheer oops that was testing purposes. I will remove it